### PR TITLE
kubeadm: add kinder leftover test jobs for k8s 1.21

### PR DESCRIPTION
**Pull request by RA489**
xref https://github.com/kubernetes/kubeadm/pull/2412

### CODE REVIEW

File: config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-kubelet-x-on-y.yaml [hunk 1]
```diff
@@ -28,12 +28,92 @@ periodics:
...
+    path_alias: k8s.io/kubeadm
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210327-170ffe2-1.21
       command:
       - runner.sh
       - "../kubeadm/kinder/ci/kinder-run.sh"
       args:
-      - "skew-kubelet-1.20-on-latest"
+      - "skew-kubelet-1.19-on-1-21"
```
Comment (neolit123) [inline config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-kubelet-x-on-y.yaml:116]:
```suggestion
      - "skew-kubelet-1.19-on-1.21"
```

File: config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-kubelet-x-on-y.yaml [hunk 2]
```diff
@@ -28,12 +28,92 @@ periodics:
     path_alias: k8s.io/kubeadm
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210402-0a23031-master
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210327-170ffe2-1.21
```
Comment (neolit123) [inline config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-kubelet-x-on-y.yaml:31]:
```suggestion
    - image: gcr.io/k8s-testimages/kubekins-e2e:v20210403-e49d2c6-master
```

**Review by neolit123 — COMMENTED** [review]
there are still some items mismatched - e.g. no need for `...kubernetes-e2e-kubeadm-kinder-kubelet-1-19-on-latest` anymore.

the kubelet-skew jobs must be the same as, `skew-kubelet-*` here:
https://github.com/RA489/kubeadm/tree/983ef68d8f94655553453d7e464bd2a92649fbd8/kinder/ci/workflows

about the kubekins image:
- `name: ci-kubernetes-e2e-kubeadm-kinder-kubelet-X-on-FOO`
- the kubekins image must be `- image: gcr.io/k8s-testimages/kubekins-e2e:v20210403-e49d2c6-FOO`
(if `foo == latest` -> the image should end up with `master` instead of `latest`.

i have a WIP tool to generate these.

**Comment by neolit123** [comment]
the rest of the changes seem fine.

File: config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-kubelet-x-on-y.yaml (hunk 1 above)
Comment (RA489) [reply config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-kubelet-x-on-y.yaml]: updated

File: config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-kubelet-x-on-y.yaml (hunk 2 above)
Comment (RA489) [reply config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-kubelet-x-on-y.yaml]: updated

**Comment by neolit123** [comment]
/lgtm
/approve

lets merge this and i will make sure the remaining problems are fixed.
cc @palnabarun

**Pull request merged**